tools/lib-alert-tree/lib_alert_tree/prometheus.py

```diff
@@ -86,6 +98,16 @@ def child_id(self):
         """A short representation of this alert, for use in annotations."""
         return f"{self.name}{{{self.format_labels()}}}"
 
+    @property
+    def child_json_path(self):
+        """A JSONPath filter expression for selecting this alert as a child.
+
+        This expression will be combined into a full JSONPath query for retrieving
+        all children of a derived alert, exposed in an annotation for consumption
+        by clients (such as UIs)."""
```

Comment:
[PEP 257 - Docstring Conventions](https://www.python.org/dev/peps/pep-0257/) recommends the following style:

```suggestion
        by clients (such as UIs).
        """
```